packages/react-resize-handle/src/hooks/useResizeHandle.ts

```diff
@@ -40,6 +40,14 @@ export type UseResizeHandleParams = {
    * Use this for localization.
    */
   getA11ValueText?: (value: number) => string;
+
+  /**
+   * Only measure relative change in size, useful to use with CSS calc() function.
+   * Example: clamp(60px, calc(20% + var(--nav-size)), 40%)
+   * This will take 20% of the parent size by default, but will allow to resize between 60px and 40%.
+   * Also, the size will still be relative in nature (20% + (X)px).
+   */
+  relative?: boolean;
```

Comment:
Let's add doc for the min and max value options that they are ignored if the user sets relative